Percy.Test/Percy.Test.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net7.0</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
why r we changing framework? Is this going to be alpha release?

Reply:
Thats framework of test project. - 7.0 is eol - its added in pr description